workloads/router-perf-v2/common.sh

```diff
@@ -104,6 +104,15 @@ tune_liveness_probe(){
   # TODO: validate liveness period and image
 }
 
+reschedule_monitoring_stack(){
+   log "Re-scheduling monitoring stack to ${1} nodes"
+   kubectl get cm -n openshift-monitoring  cluster-monitoring-config -o yaml | sed "s#kubernetes.io/.*#kubernetes.io/${1}#g" | kubectl apply -f -
```

Comment:
`kubectl` commands?

Reply:
kubectl is also available in our airflow containers, fixed anyway